package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "scrypt-shim",
+  "version": "0.0.1",
+  "description": "scrypt shim",
+  "main": "src/index.js",
+  "browser": "src/browser.js",
+  "scripts": {
+    "postinstall": "node ./scripts/postinstall.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/web3-js/scrypt-shim.git"
+  },
+  "dependencies": {
+    "scryptsy": "^2.1.0",
+    "semver": "^6.3.0"
+  },
+  "author": "",
```

Comment:
Perhaps we shouldn't leave `"author"` blank? I'm not concerned about getting credit so much, just might seem strange to folks looking at it on the registry's website, etc.

Reply:
I will add ``web3.js`` as an author if this is okay for you.